articles/clients/how-to-update-client-secret.md

```diff
@@ -5,7 +5,7 @@ crews: crew-2
 
```

Comment:
Hey Katie, this doc needs some love. There are things wrong and others missing, for example:
- no reference of how to get a Mgmt APIv2 token. It's also missing from the har https://auth0.com/docs/clients/how-to-update-client-secret#using-the-management-api-to-set-the-new-client-secret
- refers to Base64-encoded client secrets

Also the client_secret does not look good at headers/titles, it's better if it was Client Secret. Can you add a new trello card in order to review this?